### DISCUSSION TITLE
Error: "" plugin is not implemented on android

### DISCUSSION THREAD

**Discussion by jarabito11** [ionic-team/capacitor#7041] [Q&A] [answered]
Hello everyone, I am using Ionic 7 with Capacitor 5.5.1
I'm having a problem on Android with the plugins (on IOS they work correctly). The error is this:

ERROR Error: Uncaught (in promise): Error: "Preferences" plugin is not implemented on android
Error: "Preferences" plugin is not implemented on android

I have the same problem with all the plugins.

I have deleted the android folder and created it again, I have synchronized the gradles, I have tried everything but it still doesn't work.

Somebody could help me?
Thank you!!!

**Comment by jcesarmobile** [comment]
See if https://capacitorjs.com/docs/android/troubleshooting#plugin-not-implemented helps

**Reply by jarabito11** [reply]
I've already tried this and it doesn't work.
I am not migrating Capacitor, the project is Capacitor 5.
I'm not using service workers either, but I've tried putting the code on this page and it doesn't work either:
https://github.com/ionic-team/capacitor/issues/1655#issuecomment-579229390

Any ideas? I've had this error for several weeks.

**Comment by jarabito11** [comment, marked as answer]
[The error ](https://github.com/admob-plus/admob-plus/issues/594#issuecomment-1691858871)
The error is AdmobPlus Capacitor, you must uninstall or try with the 1.24.0 version

**Reply by Rufai-Ahmed** [reply]
I have this same issue, with the exact same version of capacitor and ionic, but I didn't install AdmobPlus Capacitor though

**Reply by seemsindie** [reply]
Me too @Rufai-Ahmed any luck resolving it?

**Reply by Rufai-Ahmed** [reply]
Nope

**Reply by Rufai-Ahmed** [reply]
The issue with mine was that the package.json wasn't updating properly as I was installing the dependencies. So, it's a monorepo with a general dependency, and then ionic kinda generates its package.json and the dependencies' plugin wasn't updating there
